client.go

```diff
@@ -17,7 +17,7 @@ import (
 const (
 	deprecatedSuffix = "/features"
 	clientName       = "unleash-client-go"
-	clientVersion    = "4.0.0"
+	clientVersion    = "3.9.0"
```

Comment:
should we do 3.9.1?

Reply:
I was going to do this later as part of a release, but let me double check how is this done

Reply:
```suggestion
	clientVersion    = "3.9.1"
```